net/tayga/src/etc/rc.d/opnsense-tayga

```diff
@@ -21,6 +22,16 @@ command_args="-p ${pidfile}"
 
 [ -z "$tayga_enable" ] && tayga_enable="NO"
 
+tayga_start()
+{
+        echo "starting tayga"
+        ${command} ${command_args}
+        sleep 1
+        ifconfig nat64 inet ${tayga_v4address}/32 ${tayga_v4destination}
```

Comment:
I think you mixed up the address order. `tayga_v4address` is the destination address, so it should be last.

And the interface should get an IPv6 address, too. Something like:
`ifconfig nat64 inet6 ${tayga_v6tunaddress}/128`
(This is not strictly required, but as the Tayga FAQs say: _Explicitly configuring an IPv4 address and an IPv6 address on the nat64 interface is the best way to ensure proper ICMP functionality._)

Reply:
Thanks, I'll add it. Please remember, I don't use this tool, so please bear with me :)